### PR TITLE
Allow greater control to disable DLRS globally or in specific cases

### LINKED ISSUE #1469: Interface to turn off all rollups and then cleanly turn them back on with full calculates
**Is your feature request related to a problem? Please describe.**
DLRS is awesome! Thank you!  A problem that I've encountered is that when I do mass imports of data, DLRS can interfere with the import. A few problems that I run into:

- Locking errors
- Too many rollups being scheduled for later execution which either leads to name collisions or just slows the system down generally
- APEX timeouts 
- Too many SOQL query exceptions

The solution, of course, is to turn off DLRS rollups (even scheduled ones, since those still require SOQL, etc... to put each rollup on the queue) and then turn them back on after the import is over and then to run a full recalculate on any rollups that were disabled. But this can be time consuming - if you have 20 or more rollups, the time to turn each off is frustrating as is the time to turn them back on and run recalculates. And it may be true that you just want to turn off some rollups (the ones that relate to the objects that you are inserting) - if you have a custom metadata element called "DLRS_ON" and attach that as a condition to each rollup, it will only allow for ALL rollups to be turned on or off, not individual ones.

**Describe the solution you'd like**
My ideal solution would be a set of screen flows that provide a nice interface to deactivating multiple rollups simultaneously, reactivating them, and choosing which should have full recalculates. I imagine there is backend work to do to manage which are off and on, etc...

**Describe alternatives you've considered**
I know people use custom metadata to turn off all rollups by putting that into the filters, but that solution is imperfect. First, it requires the administrator to know about this "hack", which isn't obvious to everyone especially people using a cool declarative tool like DLRS. Second, it doens't solve the problem of the aftermath - rescheduling all of the recalculations. Finally, it doesn't allow for granularity. You turn off all rollups or you keep them on. You can't have an inbetween.

**Additional context**
Thank you for your thoughtfulness with this tool - I look forward to its continued improvement.

### PR DESCRIPTION
**Pull request by aheber**
# Changes
* Allow DLRS to be disabled globally using new Custom Setting field
* Allow DLRS to be disabled globally using new Apex API method: `RollupService.bypassAll()`
* Allow individual Rollups to have a Custom Permission assigned that, when held by the user, will disable that rollup

# Issues Closed
#1469
#786
#468

### CODE REVIEW

File: dlrs/main/lwc/rollupEditor/rollupEditor.html [hunk 1]
```diff
@@ -479,6 +483,23 @@ <h3 class="slds-section__title slds-theme--shade">
             errors={errors.testCodeSeeAllData}
           ></c-rollup-editor-error>
         </lightning-layout-item>
+        <lightning-layout-item
+          size="12"
+          large-device-size="6"
+          class="slds-form-element_stacked"
+        >
+          <lightning-input
```
Comment (aheber) [inline dlrs/main/lwc/rollupEditor/rollupEditor.html:491]: @sharicarlson13 label and help text review, please :)

File: dlrs/main/objects/DeclarativeLookupRollupSummaries__c/fields/DisableDLRSGlobally__c.field-meta.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>DisableDLRSGlobally__c</fullName>
```
Comment (aheber) [inline dlrs/main/objects/DeclarativeLookupRollupSummaries__c/fields/DisableDLRSGlobally__c.field-meta.xml:3]: @sharicarlson13 FullName, description, help text, and label review, please :)

File: dlrs/main/objects/LookupRollupSummary2__mdt/fields/BypassPermissionApiName__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>BypassPermissionApiName__c</fullName>
```
Comment (aheber) [inline dlrs/main/objects/LookupRollupSummary2__mdt/fields/BypassPermissionApiName__c.field-meta.xml:3]: @sharicarlson13 fullName, helptext, and label review, please :)

**Review by octatau — DISMISSED** [review]

File: dlrs/main/lwc/rollupEditor/rollupEditor.html (hunk 1 above)
Comment (sharicarlson13) [reply dlrs/main/lwc/rollupEditor/rollupEditor.html:491]: Looks good to me

File: dlrs/main/objects/DeclarativeLookupRollupSummaries__c/fields/DisableDLRSGlobally__c.field-meta.xml (hunk 2 above)
Comment (sharicarlson13) [reply dlrs/main/objects/DeclarativeLookupRollupSummaries__c/fields/DisableDLRSGlobally__c.field-meta.xml:3]: Looks good to me

File: dlrs/main/objects/LookupRollupSummary2__mdt/fields/BypassPermissionApiName__c.field-meta.xml (hunk 3 above)
Comment (sharicarlson13) [reply dlrs/main/objects/LookupRollupSummary2__mdt/fields/BypassPermissionApiName__c.field-meta.xml:3]: Looks good to me

**Review by octatau — DISMISSED** [review]

**Comment by aheber** [comment]
@octatau could I get another review on this one? We expanded the scope of the behavior toggles to background jobs.

**Review by octatau — APPROVED** [review]

**Pull request merged**